recipes/jupyter_server_xarray_leaflet/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "jupyter_server_xarray_leaflet" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/jupyter-server-xarray-leaflet/jupyter_server_xarray_leaflet-{{ version }}.tar.gz
+  sha256: 9f742acb41278c9b71087f7d76bb62caae511b1dddd0a76df9fe0658e6992fbd
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - hatchling
+  run:
+    - python >=3.6
+    - jupyter_server >=0.2.0
+
+test:
+  imports:
+    - jupyter_server_xarray_leaflet
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/xarray-contrib/xarray_leaflet/jupyter_server
```

That URL 404's. This works:
```suggestion
  home: https://github.com/xarray-contrib/xarray_leaflet/tree/main/jupyter_server
```

---

Thanks!